<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Lab</class>
 <widget class="QMainWindow" name="Lab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>1100</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto Condensed</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Input Image</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>40</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto Condensed</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Output Image</string>
    </property>
   </widget>
   <widget class="QLabel" name="img1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>350</width>
      <height>350</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="img2">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>150</y>
      <width>350</width>
      <height>350</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="current">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>150</y>
      <width>300</width>
      <height>300</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>40</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto Condensed</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Current Transformation</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1300</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuKernel">
    <property name="title">
     <string>Operaciones con Kernel </string>
    </property>
    <addaction name="actionSobelX"/>
    <addaction name="actionSobelY"/>
   </widget>
   <widget class="QMenu" name="menuOperaciones_Logicas">
    <property name="title">
     <string>Operaciones LÃ³gicas</string>
    </property>
   </widget>
   <addaction name="menuKernel"/>
   <addaction name="menuOperaciones_Logicas"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSobelX">
   <property name="text">
    <string>Filtro de Sobel X</string>
   </property>
  </action>
  <action name="actionSobelY">
   <property name="text">
    <string>Filtro de Sobel Y</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
